### DISCUSSION TITLE
Well impressed

### DISCUSSION THREAD

**Discussion by clarkerz** [jekhokie/raspberry-noaa-v2#298] [General]
So far I'm very impressed with the pictures I've received. I'm using a NESDR SMArTee v2 and  SAWbird+ NOAA LNA. MY aerial is a DIAMOND DP-KE137 (QFH) and the only reason I am using an LNA is I have about 15M RG213 coax run. I've enabled a UFW firewall, fail2ban and also a letsencrypt cert and I run a post install install script to copy back over the default-tls file.

I am getting some noise on certain passes mainly at the top of the image and sometimes at the bottom but when they do appear they are always in the same place. I am not sure what to set the gain to as I would have thought it should be 0 but that then sets AGC so I've set it to 1.4. I am new to all this so learning all the time and am fairly pleased so far but always willing to accept any advice
Clear:
![NOAA-19-20210320-172545-MCIR](https://user-images.githubusercontent.com/9163611/111898203-87faa580-8a1c-11eb-848e-7719bf7dafbf.jpg)
![NOAA-18-20210320-102131-MCIR](https://user-images.githubusercontent.com/9163611/111898207-88933c00-8a1c-11eb-9bcb-93cc5f2f8144.jpg)
![NOAA-15-20210318-185540-MCIR](https://user-images.githubusercontent.com/9163611/111898209-892bd280-8a1c-11eb-98a4-6c68b88d30ba.jpg)

A bit of noise.:

![NOAA-18-20210320-215401-MCIR](https://user-images.githubusercontent.com/9163611/111898228-aa8cbe80-8a1c-11eb-8bb0-236b590dded0.jpg)

and this mornings :(

![NOAA-15-20210321-075422-MCIR](https://user-images.githubusercontent.com/9163611/111898536-93e76700-8a1e-11eb-93c3-a4143b76c9e7.jpg)

Am I being too fussy??

Andy

**Comment by jekhokie** [comment]
@clarkerz very nice captures! Sounds like you’re now at the ‘fine tuning’ stage. Come join us on Discord - there’s actually quite a bit of trial/effort at this stage and many of us can help you try a few things, although you might just be hitting a ‘as good as I can get it’ stage. Here’s the Discord link in case you’re interested: https://discord.gg/368C6PuC

**Reply by clarkerz** [reply]
Thanks, just joined Discord...think I might put the Pi into a  ruggidised box under the antenna or at least test that today - with and without the LNA.
